src/module-declaration.ts

```diff
@@ -53,11 +52,23 @@ export const generateModuleDeclaration = (
   // Interface Declaration
   if (newModule) {
     if (module.type !== 'Structure') {
-      if (utils.isEmitter(module)) {
+      let extendsInfo = '';
+      if (module.extends) {
+        extendsInfo = ` extends ${module.extends}`;
+      } else if (
+        utils.isEmitter(module) ||
+        (isStaticVersion &&
+          instanceModuleForStaticVersion &&
+          utils.isEmitter(instanceModuleForStaticVersion))
+      ) {
+        extendsInfo = ` extends ${isClass ? 'NodeEventEmitter' : 'NodeJS.EventEmitter'}`;
+      }
+      if (module.name.toLowerCase() === 'session' && isStaticVersion) {
+        console.log({ isStaticVersion, instanceModuleForStaticVersion, extendsInfo });
+      }
```

Comment:
@MarshallOfSound, was this debug code that got left in accidentally? Seeing the output on `e/e` now.